engine/src/main/java/org/terasology/engine/core/modes/StateMainMenu.java

```diff
@@ -155,11 +168,13 @@ public String getLoggingPhase() {
 
     @Override
     public boolean isHibernationAllowed() {
-        return true;
+        return !headless;
```

Comment:
?

Reply:
Headless mode is not needed in hibernation.

HibernationSystem lower frame rate  while game off-focus.

P.S. it is useless:
1. MainMenu don't receive any tick.
2. Seems MTE don't starts hibernationSubsystem.